services/repository/files/update.go

```diff
@@ -237,22 +237,20 @@ func ChangeRepoFiles(ctx context.Context, repo *repo_model.Repository, doer *use
 		}
 
 		for _, file := range opts.Files {
-			if err := handleCheckErrors(file, commit, opts); err != nil {
+			if err = handleCheckErrors(file, commit, opts); err != nil {
 				return nil, err
 			}
 		}
 	}
 
-	contentStore := lfs.NewContentStore()
 	for _, file := range opts.Files {
 		switch file.Operation {
 		case "create", "update", "rename":
-			if err := CreateOrUpdateFile(ctx, t, file, contentStore, repo.ID, hasOldBranch); err != nil {
+			if err = CreateUpdateRenameFile(ctx, t, file, lfs.NewContentStore(), repo.ID, hasOldBranch); err != nil {
```

Comment:
`lfs.NewContentStore()` doesn't need to be created for every file.

Reply:
Oops, mistake.